releasing/play.md

```diff
@@ -124,24 +121,28 @@ When ready:
 
 ```bash
 cd deploy
-./release --branch <branch>
+./release --project playframework --branch <branch> --tag <new-tag>
 ```
 
-Where branch is the branch of Play that you want to release, eg 2.7.x.  You will be prompted for the version
+Where branch is the branch of Play that you want to release, eg 2.7.x. `master` and `2.8.x` already use `dynver` so 
+you must use the `--tag` argument. For `2.7.x` will be prompted for the version
 number and the next version number.  For minor releases, the default should usually be appropriate.
 
 Once Play is released, you need to wait 10 minutes or so for a Maven central sync before you can perform any of
 the remaining tasks.
 
 **Verification**: You can check that the artifacts are available at Maven Central under play_\<scalaversion>.
-<https://repo1.maven.org/maven2/com/typesafe/play/>
+<https://repo1.maven.org/maven2/com/typesafe/play/>. You can see the staged artifacts here on OSS Sonatype. E.g. here is a search
+for Play 2.8.2 artifacts:
+<https://oss.sonatype.org/#nexus-search;gav~com.typesafe.play~~2.8.2~~>
+
 
 **Warning**: the play release will create a tag on the repository and push it to GitHub. That will trigger one
-or two travis jobs that are granted to fail. The reason of the failure is a circular dependency with Omnidoc.
-Play vX.Y.Z uses Omnidoc vX.Y.Z so those two travis jobs triggered by the tag and the changes in version.sbt
-will be broken until you release Omnidoc few steps down this document.
+or two travis jobs. These jobs will fail for `2.7.x` (not on other branches). The reason of the failure is a 
+circular dependency with Omnidoc. 
 
-**Warning**: If you are releasing a major version of Play which requires a new branch, for example, 2.7.0, you also need to configure MiMa before merging new features at this new branch. Ideally, do this as part of the release process.
+**Warning**: If you are releasing a major version of Play which requires a new branch, for example, 2.9.0, 
```

Comment:
Are we still branching for new releases?

Reply:
We kind of decide to not continue with this strategy. For the sake of alignment, we will keep working on master and only branch if there are breaking changes in master.

Reply:
I'll ammend. Good catch.